### DISCUSSION TITLE
Expose a button to force doorbells to ring for HomeKit automations

### DISCUSSION THREAD

**Discussion by lmaced0** [koush/scrypted#1052] [Ideas]
TL;DR: Scripted currently allows you to expose a button in HomeKit for doorbells in order to create automations. The idea here is to create a dummy button to allow rich notification (with image from the feed) on your phone and the chime on HomePods.

Hear me out... 

I currently have a Reolink doorbell and I have something in place that every time the camera sees a human it will trigger a "human detected" motion sensor in HomeKit. That's useful to build automations like turning on certain lights and also to make HomePod play a sound to get my attention.

But HomeKit is terrible at making HomePod play stuff, it will often take 30 seconds after the sensor is triggered because it's getting the sound from freaking Apple Music, dumbest thing I've ever seen.

The Doorbell chime on the other hand is instant on HomePods, plus you also get rich notifications with an image of the camera feed, which is neat.

My idea is to create a dummy button on HomeKit (the same way we already do to expose the doorbell's button for automations) to have Scrypted send a fake doorbell press to HomeKit so we can get the rich notification + the doorbell chime on our HomePods.
